pkg/operator/ceph/csi/secrets.go

```diff
@@ -105,9 +105,10 @@ func cephCSIKeyringCephFSNodeCaps() []string {
 
 func cephCSIKeyringCephFSProvisionerCaps() []string {
 	return []string{
-		"mon", "allow r",
+		"mon", "allow r, allow command 'osd blocklist'",
 		"mgr", "allow rw",
 		"osd", "allow rw tag cephfs metadata=*",
+		"mds", "allow *", // TODO: replace '*' with required permissions
```

Comment:
What mds privs are needed? Agreed we don't want to use `*`.

Reply:
We need the privs for these 2 commands particularly
ceph tell mds.0 client ls
ceph tell mds.0 client evict id=<id>

as soon as it gets confirmed by cephfs team, about the privs for these commands, I'll update it or send an updated PR. hence added TODO.

Reply:
Can at least start with `"allow rw"`? Hopefully that is sufficient until we get the feedback.

Reply:
I tried with rw, it didn't had the required permissions. 
It was popping 
Error EACCES: problem getting command descriptions from mds.0

Reply:
I see, let's go with this for now.